sky/backends/cloud_vm_ray_backend.py

```diff
@@ -4084,6 +4085,70 @@ def _execute_storage_mounts(self, handle: CloudVmRayResourceHandle,
         end = time.time()
         logger.debug(f'Storage mount sync took {end - start} seconds.')
 
+    def _execute_storage_csync(
+            self, handle: CloudVmRayResourceHandle,
+            storage_mounts: Dict[Path, storage_lib.Storage]) -> None:
+        """Executes continuous syncing on storages.
+
+        This function only runs the CSYNC daemon on the given storage
+        and the files/dirs to be copied to remote node are handled in
+        the _execute_file_mounts.
+
+        """
+        storage_mounts = {
+            path: storage_mount
+            for path, storage_mount in storage_mounts.items()
+            if storage_mount.mode == storage_lib.StorageMode.C_SYNC
+        }
+
+        if not storage_mounts:
+            return
+
+        cloud = handle.launched_resources.cloud
+        if isinstance(cloud, clouds.Local):
+            logger.warning(
+                f'{colorama.Fore.YELLOW}Sky On-prem does not support '
+                'storage syncing. No action will be taken.'
+                f'{colorama.Style.RESET_ALL}')
+            return
+
+        fore = colorama.Fore
+        style = colorama.Style
+        plural = 's' if len(storage_mounts) > 1 else ''
+        logger.info(f'{fore.CYAN}Processing {len(storage_mounts)} '
+                    f'storage continuous sync{plural}.{style.RESET_ALL}')
```

Comment:
ditto. Should we make the term in the logging message consistent, i.e., csync everywhere or `continuous sync (CSYNC)`?